README.md

```diff
@@ -1,6 +1,6 @@
 # native_updater
 
-Flutter package for prompting users to update with native dialog when there is a newer version of the app in the store.
+Flutter package for prompting users to update with a native dialog when there is a newer version of the app in the store.
```

Comment:
The idea of this project is that the user can decide whether they want to use the app store version or any version at their discretion. The package should only provide gateways to the dialogues.

Reply:
Is this correct?
"Flutter package for prompting users to update with a native dialog whether using the app store version or any version at the user's discretion."

Reply:
No, it's not correct, but I think we can get back to this description later on. Please ignore my comment and move forward to the other comments.